apps/health/lib/health/checkers/run_queue.ex

```diff
@@ -5,13 +5,14 @@ defmodule Health.Checkers.RunQueue do
   low.
   """
   require Logger
+  @max_run_queue_length 100
 
   def current do
     [run_queue: queue_size()]
   end
 
   def healthy? do
-    h? = queue_size() < 50
+    h? = queue_size() <= @max_run_queue_length
```

Comment:
suggestion: [per recent comment by @jzimbel-mbta](https://mbta.slack.com/archives/C06R4UC5A0Y/p1716392389874079?thread_ts=1716383146.205279&cid=C06R4UC5A0Y), it seems zero-arity functions are now preferred over module attributes for use as constants: https://github.com/elixir-lang/elixir/commit/19d3aeedf7a460f971dfd54d319a2d0b614ceee0#diff-59f31eea239a8bcfc94f06788b08dadc8496ce62cb74483dd0c95d9320d16f2cR136